docs/documentation/accessibility.md

```diff
@@ -0,0 +1,223 @@
+# Accessibility Guidelines
+
+According the [Strategy 2017](http://www.ilias.de/docu/goto_docu_file_5585_download.html)
+ILIAS aims to to be "Usable for Everyone" including users with such special needs.
+ILIAS therefore aims to conform WCAG (Web Content Accessibility Guidelines) 2.0 on
+conformance level A (see [https://www.w3.org/TR/WCAG20/](https://www.w3.org/TR/WCAG20/)
+). WCAGs "cover a wide range of recommendations for making Web content more accessible"
+. They claim that by "following these guidelines will make content accessible to a
+wider  range of people with  disabilities, including blindness and low vision, deafness
+and hearing loss, learning disabilities, cognitive limitations, limited movement,
+speech disabilities, photosensitivity and combinations of these". The base guideline
+concerning accessibility in ILIAS is therefore:
+
+- HTML-Output generated by ILIAS **MUST** comply with all WCAG 2.0 level A.
+
+Note: In several scenarios, developers can make not guarantees/statements, whether
+those  guidelines are met by user generated content (such as uploaded videos or files).
+However, developers SHOULD make a best effort, to support users in generating only
+content that meets the listed guidelines.
+
+The WCAG are structured by four principles that provide the foundation for Web
+accessibility: perceivable, operable, understandable, and robust. In the following
+sections we list all WCAG 2.0 level A guidelines we believe to have potential
+consequences on developing ILIAS consequences. Note that they are originate from:
+[https://www.w3.org/TR/WCAG20/](https://www.w3.org/TR/WCAG20/). If unsure on how to
```

Comment:
Maybe link at appropriate locations in the paragraph (e.g.) WCAG 2.0 instead of explicitly writing down the link.